.buildkite/pipeline-pull-request.yml

```diff
@@ -124,6 +124,28 @@ steps:
           run: integration-test-cassandra
           config: docker/buildkite/docker-compose-es7.yml
 
+  - label: ":golang: integration test with cassandra with pinot"
```

Comment:
are you planning to update pipeline-master.yml later on?

Reply:
Updated this file as well.